.github/buildEverything.py

```diff
@@ -11,14 +11,11 @@
 pioConfig = 'platformio.ini'
 
 # Configure editions
-editions = [
-    'LIGHT_EDITION_V3_2',
-    'LIGHT_EDITION_V3_3',
-    'LIGHT_EDITION_V4_0',
-    'LIGHT_EDITION_DEV_LUA',
-    'GPS_EDITION_V3_1',
-    # GPS_EDITION_DEV_ROTATED not, as it is only for testing (right now)
-]
+editions = []
+build_models = os.popen("cat platformio.ini | grep '\[\env:.*\]\'").read()
```

Comment:
I would strongly encourage you to use this instead:

```python
fh = open('platformio.ini', 'r')
for line in fh.readlines():
  if line.startswith('[env:'):
    print(line.strip())
fh.close()
```

...which is more platform-independent than your `cat | grep` solution...

Reply:
I will change it for all OS users !